rpc_jobs/rpc_jenkins_etl.yml

```diff
@@ -26,3 +26,29 @@
     jobs:
       - 'Pull-Request-Whisperer_{repo}'
 
+- project:
+    name: "rpc-jenkins-etl-dev-premerge"
+    repo_name: "rpc-jenkins-etl"
+    repo_url: "https://github.com/rcbops/rpc-jenkins-etl"
+    series: "dev"
+    branches:
+      - "dev"
+    image: "xenial"
```

Comment:
For this image, it'll also need this parameter to use nodepool which is preferred:

```
    image: "xenial":
          SLAVE_TYPE: "nodepool-ubuntu-xenial-g1-8"
```

Reply:
The edit is missing the ``:`` - currently this is not valid YAML.